fire/cli/info.py

```diff
@@ -385,7 +388,10 @@ def punkt(ident: str, obs: str, koord: str, detaljeret: bool, **kwargs) -> None:
         ident = ident.replace("GM", "G.M.", 1)
 
     try:
-        punkter = firedb.hent_punkter(ident)
+        if "%" in ident:
```

Comment:
Er det ikke mere hjemmevant at bruge `*`? Så skal der bare lige laves en hurtig replace et eller andet sted for at oversætte til SQL

Reply:
Problemet er at når man anfører '*' på kommandolinjen så vil kommandoprocesseren (bash, cmd, eller hvadnumanbruger) forsøge at matche filnavne med den struktur - og så skal man i stedet til at lære folk om escaping af jokertegn, så det transformerer problemet fra et lærings- til at "aflærings"problem. Jeg tror det er lettere at lære folk noget nyt end at lære dem workarounds

Reply:
God pointe